index.html

```diff
@@ -1825,23 +1858,73 @@ <h3 id="applying">
           <pre class="">
 {
   "icons": [
-      {
-        "src": "icon/lowres.webp",
-        "sizes": "48x48",
-        "type": "image/webp"
-      },{
-        "src": "icon/lowres",
-        "sizes": "48x48"
-      },{
-        "src": "icon/hd_hi.ico",
-        "sizes": "72x72 96x96 128x128 256x256"
-      },{
-        "src": "icon/hd_hi.svg",
-        "sizes": "257x257"
-      }]
- }
+    {
+      "src": "icon/lowres.webp",
+      "sizes": "48x48",
+      "type": "image/webp"
+    },{
+      "src": "icon/lowres",
+      "sizes": "48x48"
+    },{
+      "src": "icon/hd_hi.ico",
+      "sizes": "72x72 96x96 128x128 256x256"
+    },{
+      "src": "icon/hd_hi.svg",
+      "sizes": "257x257"
+    }
+  ]
+}
 </pre>
         </div>
+        <div class="example">
+          <p>
+            In the following example, the developer is defining <a>adaptive icons</a>
+            with a fallback for UA not supporting that.
+          </p>
+          <pre class="">
```

Comment:
Yeah, we should clean that up. We should sit down together at TPAC and sped a day doing cleanup.